### DISCUSSION TITLE
[newbie] Adding disks in Mayastor or mixing disk types

### DISCUSSION THREAD

**Discussion by mgiammarco** [openebs/openebs#3812] [Q&A]
Hi,
I am trying Mayastor, installed using microk8s. I made a pool with 3 nvme 1tb disks. It works very well but now I have two questions:
1) what happens if I now add 2tb disks in each server, can I use them to increase existing pool space?
2) can I add other disks I have (hdd) in a different pool to create another storage class?

Thanks,
Mario

**Comment by tiagolobocastro** [comment]
Hi,

1. At the moment you can only add more pools (we have limitation of 1 disk per pool). We have a roadmap item to allow for multiple disks per pool.
2. Yes you can create many pools and you can use diskpool labels to control placement: https://openebs.io/docs/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-configuration#topology-parameters

Thanks,
Tiago
